snuba/web/split.py

```diff
@@ -342,17 +350,21 @@ def execute(
             if match:
                 return None
 
-        total_col_count = len(query.get_all_referenced_columns())
-        total_ast_count = len(
-            # We need to count the number of table/column name pairs
-            # not the number of distinct Column objects in the query
-            # so to avoid counting aliased columns multiple times.
-            {
-                (col.table_name, col.column_name)
-                for col in query.get_all_ast_referenced_columns()
-            }
-        )
+        total_legacy_cols = query.get_all_referenced_columns()
```

Comment:
These seem to be caused by the legacy representation mixing literals and column names together at times while the ast tells literals apart from columns.